_posts/2020-01-20-pulp-3-GA-demos.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Pulp 3.0 GA Demos
+author: Dana Walker
+tags:
+  - 3.0
+---
+To go along with Pulp 3.0 GA, we have prepared a playlist of demos to cover features in pulpcore as
+well as various features from some of our plugins.
```

Comment:
Just pulpcore (for now). We can announce the other plugin playlists when they are ready.